plugins/products.test.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {Client4} from 'mattermost-redux/client';
+
+import testConfigureStore from 'tests/test_store';
+
+import {initializeProducts} from './products';
+
+(window as any).REMOTE_CONTAINERS = {};
```

Comment:
out of curiosity, why you need to define this?

Reply:
That's something I added a while back, and it needs to be mocked for the unit tests since it's normally injected at build time.

"Remote containers" are what Webpack calls standalone builds that are set up for module federation, and this object stores whether or not one of those containers is available. Right now, the Boards product is the only remote container we have, and it's optional because we don't require it for development.